tests/audit-scanner.bats

```diff
@@ -51,21 +53,88 @@ setup() {
     privileged_pod_uid=$(kubectl get pods nginx-privileged  -o=json | jq -r ".metadata.uid")
     local report=$(kubectl get policyreports "$privileged_pod_uid" -o json | jq -ec)
     echo "$report" | jq -e '.summary.fail == 1'
-    echo "$report" | jq -e '.summary.pass == 0'
+    echo "$report" | jq -e '.summary.pass == 1'
     echo "$report" | jq -e '[.results[] | select(.policy == "clusterwide-privileged-pods") | .result=="fail"] | all'
+    echo "$report" | jq -e '[.results[] | select(.policy == "clusterwide-safe-labels-for-pods") | .result=="pass"] | all'
+
+    unprivileged_pod_uid=$(kubectl get pods nginx-unprivileged  -o=json | jq -r ".metadata.uid")
+    local report=$(kubectl get policyreports "$unprivileged_pod_uid" -o json | jq -ec)
+    echo "$report" | jq -e '.summary.fail == 0'
+    echo "$report" | jq -e '.summary.pass == 2'
+    echo "$report" | jq -e '[.results[] | select(.policy == "clusterwide-privileged-pods") | .result=="pass"] | all'
+    echo "$report" | jq -e '[.results[] | select(.policy == "clusterwide-safe-labels-for-pods") | .result=="pass"] | all'
+
+}
+
+@test "[Audit Scanner] Delete some policies and retrigger audit scanner job" {
+    kubectl delete -f $RESOURCES_DIR/safe-labels-pods-policy.yaml
+    kubectl delete -f $RESOURCES_DIR/namespace-psa-label-enforcer-policy.yaml
+    wait_for_default_policy_server_rollout
+    
+    kubectl delete --ignore-not-found job testing --namespace $NAMESPACE
+    run kubectl create job  --from=cronjob/audit-scanner testing  --namespace $NAMESPACE
```

Comment:
please add assert_success or equivalent after run statements, otherwise errors are ignored

```suggestion
    run kubectl create job  --from=cronjob/audit-scanner testing  --namespace $NAMESPACE
    assert_success
```